### PR TITLE
Add stricter configuration rule for the project

### PR DESCRIPTION
**Pull request by brunoocasali**
https://github.com/raywenderlich/swift-style-guide

Related to #235

### CODE REVIEW

File: com.raywenderlich.swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - ${PWD}/.build
+  - ${PWD}/Demos/PerfectDemo/.build
```
Comment (brunoocasali) [inline com.raywenderlich.swiftlint.yml:6]: Unfortunately the swiftlint do not support wildcards for this use case :/

File: .swiftlint.yml [hunk 2]
```diff
@@ -1,43 +1,109 @@
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - ${PWD}/.build
+  - ${PWD}/Demos/PerfectDemo/.build
+  - ${PWD}/Demos/VaporDemo/.build
```
Comment (brunoocasali) [inline .swiftlint.yml:7]: Unfortunately the swiftlint do not support wildcards for this use case :/

Comment (bidoubiwa) [reply .swiftlint.yml:7]: Do you mean we have to specify file by file ?

Comment (brunoocasali) [reply .swiftlint.yml:7]: Not the "file" itself, but the directory, for example for `.build` we can't just use: ` - ${PWD}/**/.build/"` :(

**Comment by brunoocasali** [comment]
> Hey @brunoocasali is this still relevant?

Its is something that makes our SDK better, but this is something we can live without, so is up to you, I would like to continue working on this issue in the future :shipit:

**Comment by brunoocasali** [comment]
Unfortunately, I don't have time to work on this code anymore. Closing!

**Pull request closed without merging**